modules/mysql-flexible-server/src/variables.tf

```
@@ -30,10 +30,14 @@ variable "log_analytics_workspace_id" {
 
 # https://learn.microsoft.com/en-us/azure/azure-monitor/reference/supported-logs/microsoft-dbformysql-flexibleservers-logs
 variable "log_categories" {
+  type    = list(string)
+  default = []
+}
+
+variable "log_category_groups" {
   type = list(string)
   default = [
-    "MySqlAuditLogs",
-    "MySqlSlowLogs"
+    "allLogs"
```

Not sure how chatty it'll be in terms of LA storage costs, dunno?
```suggestion
    "audit"
```